requirements.txt

```diff
@@ -13,7 +13,7 @@ MarkupSafe==1.1.1
 multidict==4.5.2
 requests==2.22.0
 six==1.12.0
-slackclient==2.1.0
+slackclient==2.2.1
```

Comment:
we get the whole response for errors and can thus log/act on status codes: https://github.com/slackapi/python-slackclient/pull/527